editors/code/package.json

```diff
@@ -1,15 +1,15 @@
 {
     "name": "rust-analyzer",
     "displayName": "rust-analyzer",
-    "description": "A language server for the rust programming language",
+    "description": "The future of Rust support in VSCode.",
```

Comment:
Also the present hopefully 😅,

Reply:
Yeah I don't know, maybe someone has a nicer description :smile: I guess we could also go for something more 'boring', but I felt this helps a bit more to differentiate this extension from `rust-lang.rust`

Reply:
The New and Improved Rust extension!

Reply:
I'd just say "Rust language support for Visual Studio Code". And the rust-lang.rust one should say "use the other one!"